Missionframework/modules/26_cratefiller/README.md

```diff
@@ -0,0 +1,14 @@
+# KP Liberation Module Description
+
+## Example Module
+This is an example module description.
+Used as template for new modules. Make sure to replace the `example` everywhere accordingly.
```

Comment:
Still example text

Reply:
I'll do some documentation with the next sprint, so I just left it empty.